### PR TITLE
[SYCLomatic][Doc] Add CMake script migration section in Velocity-Bench/cudaSift/README.md

### PR DESCRIPTION
**Pull request by tomflinda**
Signed-off-by: chenwei.sun <chenwei.sun@intel.com>

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: third-party-programs/Velocity-Bench/cudaSift/README.md [hunk 1]
```diff
@@ -140,6 +140,64 @@ $ make -f Makefile.dpct
 
    Total workload time = 2206.28 ms
 ```
+
+### 7 Migrate CMake script and use migrated CMake script to build the SYCL version cudaSift
```
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md]:
```suggestion
### 7 Migrate CMake script and use the migrated CMake script to build the SYCL version cudaSift
```

File: third-party-programs/Velocity-Bench/cudaSift/README.md [hunk 2]
```diff
@@ -140,6 +140,64 @@ $ make -f Makefile.dpct
...
+In section 3,  using the migration command
```
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md]:
```suggestion
In section 3, using the migration command.
```

File: third-party-programs/Velocity-Bench/cudaSift/README.md (hunk 1 above)
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md]: Please merge into step3,  another option to migrate the build scripts.

File: third-party-programs/Velocity-Bench/cudaSift/README.md [hunk 3]
```diff
@@ -140,6 +140,64 @@ $ make -f Makefile.dpct
...
+
+```
+$ dpct --in-root=. -p=./build/compile_commands.json --out-root=out --gen-build-script --cuda-include-path=/usr/local/cuda/include
+```
+We have got the SYCL version cudaSift code, then we can append the option "--migrate-build-script-only" to the migration command to only migrate the CMake script, command like:
```
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md]: Don't use We/I,  to align the style.

**Review by ShengchenJ — APPROVED** [review]

**Review by AndyCHHuang — APPROVED** [review]
LGTM

File: third-party-programs/Velocity-Bench/cudaSift/README.md [hunk 4]
```diff
@@ -100,6 +101,65 @@ Now you can see the migrated files in the `out` folder as follow:
       └── matching.dp.cpp
 
    ```
+
+## 3.2 Migrate the source code and CMake build script
+In section 3.2, after running the migration command:
```
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md:106]:
```suggestion
In section 3.1, after running the migration command:
```

File: third-party-programs/Velocity-Bench/cudaSift/README.md [hunk 5]
```diff
@@ -70,6 +70,7 @@ $ ls compile_commands.json  # make sure compile_commands.json is generated
 compile_commands.json
 ```
 ### 3 Migrate the source code and build script
+## 3.1 Migrate the source code and generate build script `Makefile.dpct`
```
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md:74]: add line after L72: 
Follow section 3.1 or section 3.2 to migrate the source code and corresponding build scripts.

File: third-party-programs/Velocity-Bench/cudaSift/README.md (hunk 4 above)
Comment (zhimingwang36) [inline third-party-programs/Velocity-Bench/cudaSift/README.md]: don't ref to 3.1,  just list the code migration command again.   eg.  two step: 
1. Migrate the source code. 
2. Migrate the CMake build scripts.
then build and run.

File: third-party-programs/Velocity-Bench/cudaSift/README.md (hunk 5 above)
Comment (tomflinda) [reply third-party-programs/Velocity-Bench/cudaSift/README.md:74]: okay

File: third-party-programs/Velocity-Bench/cudaSift/README.md (hunk 4 above)
Comment (tomflinda) [reply third-party-programs/Velocity-Bench/cudaSift/README.md]: okay

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**